templates/home.phtml

```diff
@@ -141,8 +141,8 @@
             <button id="nav-mark"><?= \F3::get('lang_markread')?></button>
 
             <div id="nav-filter-wrapper">
-            <h2 id="nav-filter-title" class="nav-filter-expanded" tabindex="0"><span role="link"  aria-expanded="true"><?= \F3::get('lang_filter') ?></span></h2>
-            <ul id="nav-filter">
+            <span role="heading" aria-level="2"><h2 id="nav-filter-title" class="nav-filter-expanded" tabindex="0" role="link" aria-expanded="true"><?= \F3::get('lang_filter') ?></h2></span>
```

Comment:
> This is not valid HTML. We should make it a `<button>` inside `<h2>`.

I'll change appropriately.

Reply:
> This is not valid HTML. We should make it a `<button>` inside `<h2>`.

I've just tried that but it doesn't work.
Because the class name is coded in the js file and thus we have to use aria-expanded="true" on the heading only.
And if we do not use role="link" then aria-expanded doesn't work for headings.
Thats why I used this hack.
If you can change somethings in order to fix it, then it will be very helpful.